task_1/dmitriy.osipov/src/main/java/com/task/utils/DataUtility.java

```diff
@@ -0,0 +1,43 @@
+package com.task.utils;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+public class DataUtility {
+
+  public static String getHash(Object object) {
+    String hash = null;
+    try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
```

Comment:
It is enough to leave only the first try-with resources block, the next one's close method will be called automatically since you wrapped it in a ByteArrayOutputStream object. You should end up with only one **catch** then.